x-pack/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -144,7 +144,7 @@ export const generalSettings: RawSettingDefinition[] = [
   {
     key: 'span_frames_min_duration',
     type: 'duration',
-    validation: getDurationRt({ min: -1 }),
+    validation: getDurationRt({ min: '-1ms' }),
```

Comment:
does this need a `validationError` value as well?

Reply:
there's also `min: -1` below, is that still relevant?

Reply:
No, it's used to set the min value on `EuiFieldNumber `